codegen-luau/runtime/runtime.lua

```diff
@@ -37,12 +37,18 @@ do
 	local mul = {}
 	local div = {}
 	local neg = {}
+	local min = {}
+	local max = {}
 	local copysign = {}
 	local nearest = {}
 
 	local assert = assert
 	local math_abs = math.abs
 	local math_round = math.round
+	local math_floor = math.floor
```

Comment:
Another small change but it would be nice if these declarations were in alphabetical order.